applications/next-visit-fan-out/values-usdfdev-prompt-processing.yaml

```diff
@@ -14,6 +14,6 @@ image:
   repository: ghcr.io/lsst-dm/next_visit_fan_out
   pullPolicy: IfNotPresent
   # Overrides the image tag whose default is the chart appVersion.
-  tag: 2.5.0
+  tag: tickets-DM-48194
```

Comment:
Should I be reviewing fan out as well? I see two branches but no PR.

Reply:
No, fan-out isn't ready yet.